content/posts/2025-01-28-cli_use_cases.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: :theme/post
+title: "Network Observability On Demand Use Cases"
+description: Command line interface usage for concrete scenarios
+tags: CLI,Monitoring,Troubleshooting
+authors: [jpinsonneau]
+---
+
+# Network Observability On Demand use cases
+
+_Thanks to: Mohamed Mahmoud, Joël Takvorian, Sara Thomas and Amogh Rameshappa Devapura for reviewing_
+
+
+If you haven't read it yet, take a look at [CLI 1.8 update](./2025-01-23-cli_whats_new_1.8.md) article to list the new introduced features in release 1.8.
```

Comment:
```suggestion
If you haven't read it yet, take a look at [CLI 1.8 update](https://netobserv.io/posts/network-observability-on-demand-1-8-update/) article to list the new introduced features in release 1.8.
```

Reply:
can you try ../network-observability-on-demand-1-8-update/ ?

Reply:
```suggestion
If you haven't read it yet, take a look at [CLI 1.8 update](../network-observability-on-demand-1-8-update/) article to list the new introduced features in release 1.8.
```